RELEASE.md

```diff
@@ -1,5 +1,4 @@
-Releases
```

Comment:
I think prettier in vscode updated this automatically on me. The visual output is the same in github, but let me know if you'd rather I switch it back.

Reply:
that's fine. We should probably put in place a markdown formateur like for the rest of the code.